opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -19,6 +19,7 @@ exports.init = function(options) {
         form = options.form || section + ' form',
         $section = $(section),
         edit = options.edit || '.editBtn',
+        $edit = $(edit),
```

Comment:
I think finding the edit button when the module is loaded may be a problem for some forms like the plot detail where content/buttons can be reloaded from the server, like the plot detail page.

Not 100% sure though. @RickMohr would know more.